### PR TITLE
Make tabpy pip package

### PR DESCRIPTION
**Pull request by 0golovatyi**
### Improvements

- TabPy is pip package now
- Models are deployed with updated script

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,32 +57,29 @@ be able to work on TabPy changes:
...
+pytest
 ```
 
-## Unit Tests
+### Unit Tests
 
-TabPy has test suites for `tabpy-server` and `tabpy-tools` components.
-To run the unit tests use `pytest` which you may need to install first
-(see [https://docs.pytest.org](https://docs.pytest.org) for details):
+Unit tests suite can be exectud with the following command:
```
[the remark this answers is not in the document]
Comment (0golovatyi) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -131,32 +145,29 @@ Access-Control-Allow-Methods = GET, OPTIONS, POST
...
-pip install autopep8
+autopep8 -i tabpy-server/server_tests/test_pwd_file.py
 ```
 
-Then you can run the tool for a file. In the example below `-i`
-option tells `autopep8` to update the file. Without the option it
-outputs formatted code to the console.
+## Publishing TabPy Package
+
+Execute the following commands to build and publish new version of
```
[the remark this answers is not in the document]
Comment (0golovatyi) [reply CONTRIBUTING.md]: fixed

File: docs/server-config.md [hunk 3]
```diff
@@ -20,31 +22,132 @@
...
+# TABPY_STATIC_PATH = <package-path>/tabpy/tabpy_server/static
+
+# For how to configure TabPy authentication read
+# docs/server-config.md.
+# TABPY_PWD_FILE = /path/to/password/file.txt
+
+# To set up secure TabPy uncomment and modify the following lines.
+# Note only PEM-encoded x509 certificates are supported.
+# TABPY_TRANSFER_PROTOCOL = https
+# TABPY_CERTIFICATE_FILE = path/to/certificate/file.crt
```
Comment (jnegara) [inline docs/server-config.md]: These paths (certificate and key file) are still relative. Relative to what?

File: tabpy/tabpy_tools/rest.py [hunk 4]
```diff
@@ -84,15 +84,18 @@ def GET(self, url, data, timeout=None):
         self._remove_nones(data)
 
         logger.info(f'GET {url} with {data}')
+        print(f'GET {url} with {data}')
```
Comment (jnegara) [inline tabpy/tabpy_tools/rest.py]: print debugging leftovers?

File: .scrutinizer.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        python: 3.6
+    nodes:
+        analysis:
+            project_setup:
+                override:
+                    - pip install sklearn pandas numpy textblob nltk scipy
+            tests:
+                override:
+                    - py-scrutinizer-run
+                    -
```
[the remark this answers is not in the document]
Comment (0golovatyi) [reply .scrutinizer.yml]: Yes, this is the syntax, another example from random repo - https://github.com/thurask/bbarchivist/blob/master/.scrutinizer.yml

File: docs/server-config.md (hunk 3 above)
Comment (0golovatyi) [reply docs/server-config.md]: Updated documentation and the config.

File: tabpy/tabpy_tools/rest.py (hunk 4 above)
Comment (0golovatyi) [reply tabpy/tabpy_tools/rest.py]: Cleaned up.

**Pull request merged**